torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -3382,16 +3389,75 @@ at::Tensor interpolate(
     }
   }
 
+  float scale_factors_1 = -1.0f;
```

Comment:
Are these changes tested anywhere ?

Reply:
there is a test_upsampling_use_scale_factor test in test_nn.py

Reply:
Yep but it doesn't look like that invokes the scripted function at all

Reply:
@eellison in which file would you suggest adding a test?

Reply:
Next to all the tests here: https://github.com/pytorch/pytorch/blob/b8b6cee123343228fbb83e28b27bd0001393c173/test/test_jit.py#L16927